custom/icds_reports/templates/icds_reports/mobile/dashboard/mobile_dashboard.html

```diff
@@ -102,5 +103,20 @@
     minDuration: 700,
   });
   angular.bootstrap(document.getElementById('app'), [appName]);
+  $(function () {
+    // maintenance alert hide/show logic - lifted from hqwebapp/js/main.js
```

Comment:
this is more code than I'd normally like to copy / paste though wasn't sure exactly how to extract it in the easiest/cleanest way. open to suggestions if you have any.

Reply:
Roger, thanks. I added a [backlog item](https://trello.com/c/aJr4COdX/301-fix-duplication-of-maintenance-alerts-code) for this that I will hopefully get to in not too long.